test/test_cuda_sanitizer.py

```diff
@@ -477,14 +481,14 @@ def test_error_message(self):
                 CSAN detected a possible data race on tensor with data pointer 1
                 Access by stream 1001 during kernel:
                 schema
-                writing to argument: b
+                writing to arguments: b, and to the output
```

Comment:
Would be even cooler if we could use "argument" (singular) if there's just one. And maybe remove the colon (or else it seems that "to the output" is an element of the list of arguments)?